File: Practice(Beginner)/Python3/Two_Numbers.py
# Source: https://www.codechef.com/problems/COLOR

t = int(input())

for test in range(t):
    a, b, n = map(int,input().split(" "))
    if(n%2 == 1):
        a*=2
    print(max(a,b)//min(a,b))

File: Practice(Beginner)/C++/Two Numbers.cpp
// Source: https://www.codechef.com/problems/COLOR

#include <bits/stdc++.h>
using namespace std;
int main(){
    int t;
    cin >> t;
    while (t--){
        long long int a, b, n;
        cin >> a >> b >> n;
        if(n % 2 == 1){
            a *= 2;
        }
        cout << max(a,b)/min(a,b) << endl;
    }
}